billing-tests/build.gradle

```diff
@@ -5,6 +5,11 @@ apply plugin: 'com.jaredsburrows.license'
 apply plugin: 'com.mapbox.maps.token'
 apply from: "../gradle/ktlint.gradle"
 
+def mapboxBillingTestsApiToken = System.getenv('MAPBOX_BILLING_TESTS_ACCESS_TOKEN') ?: System.getenv('MAPBOX_ACCESS_TOKEN')
+if (mapboxBillingTestsApiToken == null) {
+    throw new Exception("Billing tests token is not specified.")
```

Comment:
Does it fail during configuration even if a person isn't interested in running billing tests?

Reply:
maybe we should fail tests instead of build if the token isn't provided?

Reply:
Good catch, thank you, fixed!